app/dao/jobs_dao.py

```diff
@@ -129,16 +129,15 @@ def dao_update_job(job):
     db.session.commit()
 
 
-def dao_get_jobs_older_than_data_retention(notification_types):
+def dao_get_jobs_older_than_data_retention(notification_types, limit=None):
```

Comment:
Do we need to change [`remove_transformed_dvla_files`](https://github.com/cds-snc/notification-api/blob/main/app/celery/nightly_tasks.py#L196)? Are we even running this [task](https://github.com/cds-snc/notification-api/blob/main/app/config.py#L479)?

Reply:
That's old letter code that apparently [we're still running](https://github.com/cds-snc/notification-api/blob/bfadea3bb7ba83cf57d3d206e907b01bfa9239ef/app/config.py#L479-L482) but should just be doing nothing